templates/task-sc.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
```

Comment:
Make sure the [formatting is correct](https://redhat-internal.slack.com/archives/C04J0SVUDTL/p1679555852819269), please.